Classes/Controller/AuthenticationController.php

```diff
@@ -31,6 +31,7 @@
 use TYPO3\CMS\Extbase\Reflection\ObjectAccess;
 use TYPO3\CMS\Extbase\Security\Cryptography\HashService;
 use TYPO3\CMS\Extbase\SignalSlot\Dispatcher;
++use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
```

Comment:
```suggestion
use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
```

Reply:
Can you fix this typo and push again? This should also trigger CI which was disabled in the meantime.

Reply:
Oh nooo, sorry my mistake and fixed now.